core/src/test/scala/unit/kafka/raft/RaftManagerTest.scala

```diff
@@ -178,7 +178,7 @@ class RaftManagerTest {
   }
 
   private def fileLocked(path: Path): Boolean = {
-    TestUtils.resource(FileChannel.open(path, StandardOpenOption.WRITE)) { channel =>
+    TestUtils.resource(FileChannel.open(path, StandardOpenOption.CREATE, StandardOpenOption.WRITE)) { channel =>
```

Comment:
@FrankYang0529 , could you explain more why we need this change? Thanks.

Reply:
We remove the `.lock` file when `RaftManager#shutdown`. In this case, `FileChannel` can't open a nonexistent file and will throw `NoSuchFileException`, so I add `StandardOpenOption.CREATE` to avoid the error.

Reply:
Thanks for the explanation!